### ISSUE TITLE
Doesn't build with OTP 23

### ISSUE THREAD

**Issue by weiss** [tsloughter/rebar3_run#10]
The plugin doesn't compile when using Erlang/OTP 23:

```
make: Entering directory '/tmp/myrel/_build/default/plugins/rebar3_run/c_src'
cc -O3 -std=c99 -finline-functions -Wall -Wmissing-prototypes -fPIC -I /home/xmpp/opt/erlang/lib/erlang/erts-11.0.2/include/ -I /home/xmpp/opt/erlang/lib/erlang/lib/erl_interface-4.0/include  -c -o /tmp/myrel/_build/default/plugins/rebar3_run/c_src/rebar3_run.o /tmp/myrel/_build/default/plugins/rebar3_run/c_src/rebar3_run.c
cc /tmp/myrel/_build/default/plugins/rebar3_run/c_src/rebar3_run.o -shared -L /home/xmpp/opt/erlang/lib/erlang/lib/erl_interface-4.0/lib -lerl_interface -lei -o /tmp/myrel/_build/default/plugins/rebar3_run/c_src/../priv/rebar3_run.so
/usr/bin/ld: cannot find -lerl_interface
collect2: error: ld returned 1 exit status
make: *** [Makefile:59: /tmp/myrel/_build/default/plugins/rebar3_run/c_src/../priv/rebar3_run.so] Error 1
make: Leaving directory '/tmp/myrel/_build/default/plugins/rebar3_run/c_src'
===> Hook for compile failed!

===> Plugin rebar3_run not available. It will not be used.
```

**Comment by tsloughter** [comment]
Thanks, the makefile is wrong on 23. I guess I'll want to switch to the `pc` plugin these days to make that easier.

**Comment by tsloughter** [comment]
Fixed by #11
